pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -115,18 +137,26 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 		ResyncPeriod: m.ResyncPeriod,
 		DomainSuffix: m.DomainSuffix,
 	}
+	webhookConfigName := strings.ReplaceAll(validationWebhookConfigNameTemplate, validationWebhookConfigNameTemplateVar, m.secretNamespace)
 	if m.fetchCaRoot != nil {
 		nc := NewNamespaceController(m.fetchCaRoot, opts, clientset)
 		go nc.Run(stopCh)
+		go util.PatchCertLoop(injectionWebhookConfigName, webhookName, m.caBundlePath, clientset, stopCh)
```

Comment:
In the single cluster mode `injectionWebhookConfigName` is an environment variable. Seems like maybe we should use that as well, otherwise multi-revision will not work

Reply:
injectionWebhookConfigName is defined in pilot/pkg/bootstrap/sidecarinjector.go. I have not found a proper way to reuse it in pilot/pkg/serviceregistry/kube/controller/multicluster.go. Any suggestions for this?